ServerlessLibraryUI/src/components/AppItem.vue

```diff
@@ -296,6 +296,11 @@ export default {
     font-weight: 600;
   }
 
+  &__deploybtn:disabled {
+    color: rgba(127, 127, 127, 0.7);
```

Comment:
Looks good. 
I presume this'll need to look something like this, unless you have taken this from another disabled component.

https://developer.microsoft.com/en-us/fabric#/components/button
![image](https://user-images.githubusercontent.com/2650941/53769053-960a1780-3e8f-11e9-8651-4084a916dfff.png)

Reply:
I've taken it from Azure portal:

![image](https://user-images.githubusercontent.com/45184761/53769358-85a66c80-3e90-11e9-8455-3cc1dd9f4325.png)

Reply:
Thats good.